lib/item-comments.js

```diff
@@ -0,0 +1,22 @@
+var Sequelize = require('sequelize');
+
+module.exports = function(db) {
+  var ItemComments = db.define('item_comments', {
+    id: {
+      type: Sequelize.INTEGER,
+      autoIncrement: true,
+      primaryKey: true
+    },
+    body: {
+      type: Sequelize.TEXT
+    },
+    point: {
+      type: Sequelize.GEOMETRY('POINT')
+    },
+    user: {
```

Comment:
It looks like we use `createdBy` in the item table definition. Does it make sense to keep these consistent?

Reply:
Similar to the `body`, do we want to disallow a null value to be stored?